x11/homebank/Portfile

```diff
@@ -3,7 +3,8 @@
 PortSystem          1.0
 
 name                homebank
-version             5.2.2
+version             5.2.4
+revision            1
```

Comment:
The `revision` needs to be set to 0 when increasing the `version`.

Reply:
I have remove ligne revision in commit 4b77a7bca41702e5e2393a00437dd52bca360794